js/ui/components/clubhouse.js

```diff
@@ -217,6 +239,10 @@ var ClubhouseButtonManager = new Lang.Class({
         this._clubhouseWindowActor = null;
     },
 
+    startPulsingIcon: function() {
```

Comment:
This method has a generic name, yet it refers to the open button. So I'd call it `suggestOpen` or `setOpenButtonHighlight(true|false)`.